website/docs/reference/resource-properties/constraints.md

```diff
@@ -68,9 +66,77 @@ models:
           - type: ...
 ```
 
+In [dbt Cloud Versionless](/docs/dbt-versions/upgrade-dbt-version-in-cloud#versionless), you can use the improved syntax to define foreign keys, which uses `ref`. This feature will be available in the upcoming 1.9 release. For more information, update the documentation version to 1.9 and refer to the [following section](/reference/resource-properties/constraints#whats-better-about-this-syntax).
+
 </File>
 
+</VersionBlock>
+
+<VersionBlock firstVersion="1.9">
```

Comment:
just reiterating the [same comment here](https://github.com/dbt-labs/docs.getdbt.com/pull/6105/files#r1768476630) - i don't think we should version block this 'defining constraints' section since the content is the exact same for v1.9+ and v1.8 and lower. so to avoid manually updating both contents, i recommend using version blocks on the actual content / code snippets:

![Screenshot 2024-09-24 at 12 27 15](https://github.com/user-attachments/assets/9fcac431-fa34-463e-a42b-3c11860d826c)